MANUAL_GCP_DEPLOYMENT.md

```diff
@@ -0,0 +1,18 @@
+# Deployment Guide for membrane-backend
+
+This guide outlines the steps to deploy manually membrane-backend to Google Cloud Run.
+
+## Steps
+ 1. Run the script from DevOps repo available [here](https://github.com/ai-cfia/devops)
```

Comment:
so vague... 

1. Setup your GCP environment. You can use the instructions [GCP Project Setup Guide](https://github.com/ai-cfia/devops/blob/main/gcp-setup-script/gcp-project-setup-guide.md)

Reply:
done